buildifier/buildifier.go

```diff
@@ -66,12 +68,13 @@ func stringList(name, help string) func() []string {
 }
 
 func usage() {
-	fmt.Fprintf(os.Stderr, `usage: buildifier [-d] [-v] [-mode=mode] [-lint=lint_mode] [-path=path] [files...]
+	fmt.Fprintf(os.Stderr, `usage: buildifier [-d] [-v] [-diff=diff_program] [-mode=mode] [-lint=lint_mode] [-path=path] [files...]
```

Comment:
Please add the new `-multi_diff` flag as well.

```suggestion
	fmt.Fprintf(os.Stderr, `usage: buildifier [-d] [-v] [-diff=diff_program] [-multi_diff] [-mode=mode] [-lint=lint_mode] [-path=path] [files...]
```

Reply:
I uploaded a new version of the commit with this change.  I also changed the flag name to
"diff_command" since some people will want to set it to "diff -u" ... which seems better to
refer to as the diff command to run.

I'm not now changing the default behavior.  I can remove the environment variables in a
second commit ... maybe after some time of warning that they are deprecated.